build-aux/app.drey.Biblioteca.Devel.json

```
@@ -40,6 +39,10 @@
           "type": "dir",
           "path": ".."
         }
+      ],
+      "post-install": [
+        "cp -a /usr/share/runtime/docs/doc/. /app/share/doc/",
+        "cp -a /usr/share/runtime/docs/gtk-doc/html/ /app/share/doc/"
```

```suggestion
        "cp -a /usr/share/runtime/docs/gtk-doc/html/. /app/share/doc/"
```
I think it's not copying it, but after adding `.` I get ``cp: cannot overwrite directory '/app/share/doc/./appstream' with non-directory``

---

Oops my bad it works now. I did test it as is but I got repeating entries in the sidebar.